### ISSUE TITLE
[D365 Shell & Portal][Screen reader  – PowerAppsPlayer-Web  – Filter Pane] Narrator reads 'Group' when focus moves to whole 'Failed to save preferences' pop up

### ISSUE THREAD

**Issue by sonimishra** [reactjs/react-modal#731]
### Summary:
User who depends upon screen reader will not be able to get pop up Title name if narrator does not read the name of pop up appear on the screen. 

### Steps to reproduce:

1. Navigate to https://home.dynamics.com/ and Login using valid Account
2. Traverse to the "Filter" Button.
3. Filters Section is opened.
4. Select/unselect any checkbox . observe that 'Failed to Save preferences' pop up is appear . 
5. Narrator focus moves to whole pop up appear on the screen.
6. Verify that , narrator reads 'Group' when focus is on whole pop up.

###Actual behavior : 
Narrator reads 'Group' when focus moves to whole 'Failed to save preferences' pop up. 

### Expected behavior:
Narrator should not read 'Group' when focus moves to whole 'Failed to save preferences' pop up. Narrator should read the title of the pop up i.e "Failed to save preferences" .

### Additional notes:

Issue does not exists under Chrome + JAWS.
 
MAS Reference:
MAS 4.2.1 - Object Information (33A)
![mas4 2 1_narratorreads group](https://user-images.githubusercontent.com/30456910/51658311-96301100-1fcd-11e9-8182-ba2903b6e2e9.jpg)

**Comment by sonimishra** [comment]
there is actually another place where you can reproduce this issue:
Go to home.dynamics.com
Tab until "Get more apps", hit "Enter"


This will also open a react-modal dialog, the focus goes onto the dialog, and it's reading the "Group" as well.

demo: https://codepen.io/claydiffrient/pen/KNjVrG so looks like the issue on their side is still not fixed yet.

**Comment by diasbruno** [comment]
Hi @sonimishra, for now, I can only recommend reading react-modal's documentation on [accessibility](http://reactcommunity.org/react-modal/accessibility/). If more information can be provided, we can further inspect what's going on. Thanks.

**Comment by diasbruno** [comment]
@sonimishra, I'll close this issue, because it's hard to say if this is an issue with react-modal or the application. In any case, please let me know if there is something we can do on our side. 

Thanks.

**Comment by SummerLi** [comment]
Hi @diasbruno, this is not an issue on the application side, but a bug on react-modal side, I can reproduce this issue with the react-modal official demo. 

Repro steps:
1. Open the react-modal demo with Edge browser: https://codepen.io/claydiffrient/pen/KNjVrG
2. Open "Narrator" screen reader
3. Click on the "Trigger Modal" button
4. Narrator reads: "onRequestClose Example **Group**"

So can you please re-open this issue and take a look?

**Comment by sonimishra** [comment]
As per above comment from @SummerLi , @diasbruno  Please reopen this issue for the fix.

**Comment by diasbruno** [comment]
@sonimishra @SummerLi Can you please check if this sandbox has the correct behavior?

[codesandbox.io - n0vl89vr90](https://codesandbox.io/s/n0vl89vr90)

**Comment by sonimishra** [comment]
Hi @diasbruno,

We have verified the sandbox(i.e. "codesandbox.io - n0vl89vr90") and the behavior seems fine to us since it is passing the correct information for the dialog box thus opened.

**Comment by diasbruno** [comment]
Sorry, close by mistake.

That's great. So, this is something that should be fixed on the application-side, but do you think there is something we can do? Like improve the documentation...
